### ISSUE TITLE
Reconfigure: TypeError: rpc() got multiple values for keyword argument 'exchange'

### ISSUE THREAD

**Issue by tilsche** [metricq/metricq-wizard-backend#9]
```
Nov 05 14:56:36 igel gunicorn[127849]:   File "/home/service/metricq-wizard-backend/app/api/views/client.py", line 54, in reconfigure_client
Nov 05 14:56:36 igel gunicorn[127849]:     await configurator.reconfigure_client(client_id=client_id)
Nov 05 14:56:36 igel gunicorn[127849]:   File "/home/service/metricq-wizard-backend/app/metricq/configurator.py", line 241, in reconfigure_client
Nov 05 14:56:36 igel gunicorn[127849]:     await self.rpc(
Nov 05 14:56:36 igel gunicorn[127849]:   File "/home/service/envs/wizard-backend/lib/python3.8/site-packages/metricq/client.py", line 108, in rpc
Nov 05 14:56:36 igel gunicorn[127849]:     return await super().rpc(
Nov 05 14:56:36 igel gunicorn[127849]: TypeError: rpc() got multiple values for keyword argument 'exchange'
```

How did that ever work? It's been like this for months?

https://github.com/metricq/metricq-wizard-backend/blame/master/app/metricq/configurator.py#L234-L240
https://github.com/metricq/metricq-python/blame/master/metricq/client.py#L108-L114

**Comment by kinnarr** [comment]
This bug was introduced by replacing the custom `ManagementAgent` class with metricqs `Client` class and should be fixed in 4107349fe16a50395a0801af11606847114a4696

**Comment by kinnarr** [comment]
4107349fe16a50395a0801af11606847114a4696 fixes only 1 occurence 🤦
